docs/AccelNNPAHowToUseAndTest.md

```diff
@@ -0,0 +1,29 @@
+<!--- SPDX-License-Identifier: Apache-2.0 -->
+
+# Build and test for Accelerator NNPA
+
+## Build
+
+The following CMake option is required to build onnx-mlir for NNPA. Add this option in [build command](https://github.com/onnx/onnx-mlir/blob/main/docs/BuildOnLinuxOSX.md/#build)
```

Comment:
`. Add this option in` => ` and this option is in`

Reply:
Done. Thanks!